ci-operator/jobs/openshift-kni/eco-ci-cd-public/openshift-kni-eco-ci-cd-public-main-periodics.yaml

```diff
@@ -0,0 +1,176 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: '* 1,9,17 * * *'
```

Comment:
Hi @ccardenosa 
1. The single quote is not needed here. 
2. Also, the scheduling is `every minute past hour 1, 9, and 17`, is that needed? 
It will put some pressure on the Slack channel.

Reply:
Thanks for your comments.

1) Ok
2) I have already lowered the cadence to three times a day. The plan is to reduce it further, when we see that it is more stable. For now I prefer to have it run several times a day, to maximize the chances of having valid runs.